news/2017-11-28-OWC-Round-of-16-recap.md

```diff
@@ -0,0 +1,43 @@
+
+![](\wiki\shared\news\2017-11-29-osu-world-cup-round-of-16-recap\Hong_Kong_v_Netherlands.png)
+
+(Too close for comfort!)
+
+In the Denmark-Germany match-up, a noteworthy name is [raser1234](https://osu.ppy.sh/users/2527887) of the Danish roster, who topped his team’s scoreboard in almost all maps. However, his lone efforts were not enough as the battle-hardened German players were able to summon multiple carries to defend and counteract accordingly. [Dustice](https://osu.ppy.sh/users/754565), [Neliel](https://osu.ppy.sh/users/1500305), and [imagaK](https://osu.ppy.sh/users/2022445) maintained composure and did what they would do in their best conditions – click the circles, to the beat. While the Danish spears were sharp and pointy, the German ironclad wall proved to be too hard to penetrate. As the last note of [Kyouran Hey Kids!!](http://osu.ppy.sh/b/815857&m=0) gets clicked, the match came to a win of 6-3 in favour of Germany.
+
+![](\wiki\shared\news\2017-11-29-osu-world-cup-round-of-16-recap\Denmark_v_Germany.png)
+
+(Dustice, imagaK, and Neliel – the fearless “Deutschen Stahl” trio)
+
+The osu! rivalry between Brazil and Canada is a beautiful one as well. Both teams often scrim with each other for different occasions, and therefore they know each other’s strengths and weaknesses. This match-up, therefore, became a battle of out-consisting the other team, especially when the mappool difficulty was still manageable by both sides. Canada’s roster has no “lone star carry” and instead features players wielding a wide range of skills to complement each other. [Azer](https://osu.ppy.sh/users/2155578)'s reading skills, [MiruHong](https://osu.ppy.sh/users/2866814)'s rhythmic mastery, [Ignite](https://osu.ppy.sh/users/3122948)’s speed, and [FunOrange](https://osu.ppy.sh/users/2051389)’s consistency – they are a versatile and well-rounded roster. Meanwhile, Brazil does have veteran carries like [MouseEasy](https://osu.ppy.sh/users/1558603) and [Mystia](https://osu.ppy.sh/users/4277702), but they are not alone as the rest of the roster has shown to be able to keep up and support their main powerhouses. The direct face-to-face clash came down to Brazil winning 6-3 through displaying the better consistency.
+
+![](\wiki\shared\news\2017-11-29-osu-world-cup-round-of-16-recap\Canada_v_Brazil.png)
+
+(MouseEasy demonstrating he's also SSEasy)
+
+Nonetheless, we are not saying good-bye to anyone just yet! Do remember that OWC is double-elimination, meaning all teams that make out of Groups get a second chance. Next week, we shall witness more intense circle-clicking action from these players. Some will reach for definitive victories, and some will fight to remain within the tournament. Don’t miss out! We will see you at our stream over on [Twitch](http://www.twitch.tv/osulive)! 
```

Comment:
missing dash signoff ( - Evrien, etc) except use a proper em-dash

new paragraph We will see you

maybe include information about this weekend's dates and a link to the wiki article on how to access them?